From 8dea23a2b6dae52267b3a969e715d3f0753acf47 Mon Sep 17 00:00:00 2001
From: Florian Fainelli <florian.fainelli@telecomint.eu>
Date: Wed, 10 Oct 2007 14:54:11 +0200
Subject: [PATCH] [PATCH] Add missing generic GPIO support for au1000
With the generic GPIO support for au1000, we do not
select it in the kernel configuration.

Signed-off-by: Florian Fainelli <florian.fainelli@telecomint.eu>
---
 arch/mips/au1000/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

--- a/arch/mips/au1000/Kconfig
+++ b/arch/mips/au1000/Kconfig
@@ -135,3 +135,4 @@ config SOC_AU1X00
 	select SYS_SUPPORTS_32BIT_KERNEL
 	select SYS_SUPPORTS_APM_EMULATION
 	select SYS_SUPPORTS_KGDB
+	select GENERIC_GPIO
